index.js

```diff
@@ -549,6 +602,16 @@ function getRedirectUrl (url) {
   }
 }
 
+function generateFileHash (filePath) {
+  const hashSum = crypto.createHash('md5')
```

Comment:
Is md5 the fastest hashing algorithm in node?

Reply:
Good question, I found for stuff not security related, either MD5 or SHA-1 is good and fast(est?)

But would love to learn if there's something else that's faster that can be used for the same purpose